katalog/nginx/bases/controller/deployment.yml

```diff
@@ -26,9 +26,9 @@ spec:
             - --https-port=8443
           securityContext:
             capabilities:
-                drop:
+              drop:
                 - ALL
-                add:
+              add:
```

Comment:
Since now the controller is listening on unprivileged ports, can we drop `NET_BIND_SERVICE` capability?

Reply:
I've never tried without it. Let's do it ;)

Reply:
That's not work without the `NET_BIND_SERVICE`:

```
[dumb-init] /nginx-ingress-controller: Operation not permitted
```

Reply:
😭